python/cusignal/radartools/radartools.py

```diff
@@ -172,7 +172,7 @@ def ambgfun(x, fs, prf, y=None, cut='2d', cutValue=0):
         for col in range(v.shape[1]):
             v_shift[:, col] = cp.roll(v[:, col], -col, axis=0)
         ynorm = cp.tile(ynorm, (len_seq - 1, 1))
-        amf = nfreq*cp.abs(cp.fft.fftshift(
+        amf = nfreq * cp.abs(cp.fft.fftshift(
             cp.fft.ifft(ynorm * cp.conj(v_shift), nfreq, axis=1), axes=1))
```

Comment:
Do you really need `cp.conj()`??

Reply:
Yes, it's part of the algorithm and needed for complex valued input, which hasn't been checked for correctness yet.